vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/DashboardItemMoveModeChangedEvent.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See {@literal <https://vaadin.com/commercial-license-and-service-terms>} for the full
+ * license.
+ */
+package com.vaadin.flow.component.dashboard;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.ComponentEvent;
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.DomEvent;
+import com.vaadin.flow.component.EventData;
+
+/**
+ * Widget or section move mode state changed event of {@link Dashboard}.
+ *
+ * @author Vaadin Ltd.
+ * @see Dashboard#addItemMoveModeChangedListener(ComponentEventListener)
+ */
+@DomEvent("dashboard-item-move-mode-changed")
+public class DashboardItemMoveModeChangedEvent
```

Comment:
Took me a while to understand what this does. I'm not sure what the use case is?

Reply:
Those listeners are mostly there to cover some accessibility requirements. I would expect not many users to use this otherwise.

Reply:
OK thanks